src/components/organization/organization.repository.ts

```diff
@@ -46,11 +45,9 @@ export class OrganizationRepository extends DtoRepository<
     return await query.first();
   }
 
-  async update(
-    existing: Organization,
-    changes: ChangesOf<Organization, UpdateOrganization>,
-  ) {
-    return await this.updateProperties(existing, changes);
+  async update(changes: UpdateOrganization, session: Session) {
+    await this.updateProperties({ id: changes.id }, changes);
```

Comment:
@CarsonF - Struggling to figure out the types here. It does not like `changes` on this line: 
TS2345: Argument of type `UpdateOrganization` is not assignable to parameter of type `DbChanges<Organization>`
Type `UpdateOrganization` is not assignable to type
```
Partial<Record<'id' | '__typename' | 'createdAt' | 'canDelete' | 'scope', never>>
```
Types of property `id` are incompatible.
Type `ID` is not assignable to type `undefined`

Reply:
It's say you are passing `id` in as a change and you should not be. You can spread it out of the object.
```ts
  async update(input: UpdateOrganization, session: Session) {
    const { id, ...changes } = input;
    await this.updateProperties({ id }, changes);
    return await this.readOne(id, session);
  }
```